rpgsaga/saga/src/art.ts

```diff
@@ -0,0 +1,20 @@
+export abstract class Art {
+    title: string;
+    year1: number;
+    director: string;
+    abstract toString(): string;
+
+    constructor(title: string, year: number, director: string) {
+        this.title = title;
+        this.year1 = year;
```

Comment:
вот тут как раз надо 
```suggestion
        this.year = year;
```
чтобы вызвался set